app/api/business/validators/supplier_validator.py

```diff
@@ -230,6 +231,43 @@ def __validate_document(self, documents, name, now, has_expiry=True):
 
         return errors
 
+    def validate_recruiter(self):
+        errors = []
+        recruiter = self.supplier.data.get('recruiter')
+
+        if recruiter and (recruiter == 'yes' or recruiter == 'both'):
+            labour_hire = self.supplier.data.get('labourHire', {})
+            now = pendulum.now('Australia/Canberra').date()
+            for state, state_value in labour_hire.iteritems():
+                if not state_value:
+                    continue
+                licence_number = state_value.get('licenceNumber')
+                expiry = state_value.get('expiry')
+                if (licence_number and not expiry) or (not licence_number and expiry):
+                    errors.append({
+                        'message': 'Licence number and expiry must be both filled for {}'.format(state.upper()),
```

Comment:
Might the user be confused with this error message?  If their licence number is correct but they forgot to add the expiry date or vice versa, they might think they got both of the fields wrong.